### DISCUSSION TITLE
How could tell a exe file is compressed or not like 7z or zip?

### DISCUSSION THREAD

**Discussion by snowman** [ScoopInstaller/Extras#7861] [Q&A] [answered]
In some apps' manifests, there appears URL ends with `program.exe#/dl.7z`

> Scoop will download program.exe but save it as dl.7z, which will then be extracted automatically with 7-Zip. 

But how to know the `program.exe` is compressed or not when writing a new `manifest`?

**Comment by rashil2000** [comment, marked as answer]
Right click exe -> Open with 7z. That's how I check it.

**Reply by snowman** [reply]
does `#/dl.7z` or `#/dl.zip` matter? choose one of them is okay?

**Reply by rashil2000** [reply]
Both should work. 

7z is better because it's going to be opened by 7zip after all. If we put zip then Explorer will think it's archive.
